External/FEXCore/Source/Utils/Allocator.cpp

```diff
@@ -91,6 +92,44 @@ namespace FEXCore::Allocator {
     return Result;
   }
 
+  void *DisableSBRKAllocations() {
```

Comment:
Should document somewhere *why* this is useful, and what effect it will have on glibc. IIUC we're effectively making brk/sbrk always fail, so why doesn't this break anything? Does glibc have a fallback path if sbrk doesn't work?